src/components/send/SendAssetList.js

```diff
@@ -80,9 +80,18 @@ export default class SendAssetList extends React.Component {
       assets.pop(); // removes not needed coin divider
     }
 
+    //check for placeholder ETH & remove
+    assets[0]?.isPlaceholder ? assets.shift() : null;
     const visibleAssetsLength = assets.length;
+
     this.data = assets;
+    console.log(this.data);
+    console.log(this.data.length);
     if (smallBalances.assets.length > 0) {
+      //check for placeholder ETH & remove
+      smallBalances.asset = smallBalances.assets.filter(
```

Comment:
did you mean smallBalances.assets on the left hand side? (not smallBalances.asset)